src/controller.js

```diff
@@ -1,3 +1,5 @@
+const twitterBearToken = require('../config').twitter.bearToken;
```

Comment:
typo. bearerToken

Reply:
![image](https://user-images.githubusercontent.com/16966368/112390438-8eef1580-8cb3-11eb-9109-3541f9264cd2.png)

Reply:
Fixed it. Sorry bearer, hello bear